frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -46,18 +47,31 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
 
 const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions, service }) => (
   <>
-    <SidebarSectionHeading text="Revisions" className="revision-overview-list">
-      {/* add extra check, if sidebar is opened for a knative deployment */}
-      {service.kind === ServiceModel.kind && (
-        <Button
-          variant="secondry"
-          onClick={() => trafficModalLauncher({ obj: service })}
-          disabled={!(revisions && revisions.length)}
-        >
-          Set Traffic Distribution
-        </Button>
-      )}
-    </SidebarSectionHeading>
+    <Flex
+      className="revision-overview-flex"
+      breakpointMods={
+        [
+          { modifier: 'justify-content-flex-space-between' },
```

Comment:
@vikram-raj Does this value `'justify-content-flex-space-between'` works? I think this is invalid and valid is `'justify-content-space-between'`. Please look at https://github.com/patternfly/patternfly-react/issues/3244

Reply:
Yup, This value is invalid and doesn't work. you get undefined with that value.
![Screenshot from 2019-11-12 17-58-46](https://user-images.githubusercontent.com/9278015/68671777-1e3fd100-0576-11ea-9c01-25ffad42f223.png)

![Screenshot from 2019-11-12 17-57-29](https://user-images.githubusercontent.com/9278015/68671699-ee90c900-0575-11ea-8113-69b67b70191d.png)

Reply:
@sahil143 I was trying out this option and forget to change it back. My bad.

Reply:
And ```justify-content-space-between``` is not present in ```FlexBreakpointMod``` type. https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/layouts/Flex/FlexUtils.tsx#L5

Reply:
yes, that's a bug, check out docs for flex. Docs mention `justify-content-space-between` which is correct.

Reply:
Change this to ```justify-content-space-between```.